es/The-Manual/Developing-with-CakePHP/Installation.rst

```diff
@@ -272,7 +272,7 @@ específico de un usuario o del *site*).
    En muchos servicios de hosting (GoDaddy, 1and1), tu servidor web está
    realmente siendo servido desde un directorio de usuario que ya
    utiliza mod\_rewrite. Si estás instalando CakePHP en un directorio de
-   usuario (http://ejemplo.com/~nombreusuario/cakephp), o en cualquier
+   usuario (https://ejemplo.com/~nombreusuario/cakephp), o en cualquier
```

Comment:
This link appears to be invalid. Should we remove it? @markstory

Reply:
Yeah we should remove them.

Reply:
remove